pootle/apps/contact/views.py

```diff
@@ -47,7 +47,8 @@ def get_initial(self):
 
         return initial
 
-    def get_success_url(self):
+    @staticmethod
```

Comment:
Please undo this since it is extending a class that didn't had this as a staticmethod.